transport/http_json/http_json.go

```diff
@@ -313,3 +321,22 @@ func (h *APIHandler) writeError(ctx context.Context, w http.ResponseWriter, erro
 	}
 	http.Error(w, error, code)
 }
+
+func (h *APIHandler) IsDebug(req *http.Request) bool {
+	return req.FormValue("debug") == "true"
+}
+
+func (h *APIHandler) GetDebugFromContext(ctx context.Context) (*debug.Debug, bool) {
```

Comment:
Please implement functions that control debug in gorpc root package (not in transport). 
For example transport undependable cache controlling: https://github.com/sergei-svistunov/gorpc/blob/master/transport/cache/context.go.